### ISSUE TITLE
google drive file download access denied

### ISSUE THREAD

**Issue by artificialnouveau** [anantarb/triplanenet#10]
When I run:
`source ./scripts/install_deps.sh`

I get the following errors:
```
python -c "import gdown; gdown.download('https://drive.google.com/uc?id=1BlDBB4dLLrlN3cJhVL4nmrd_g6Jx6uP0', 'checkpoints/pretrained/epoch_20.pth', quiet=False)"
Access denied with the following error:

        Cannot retrieve the public link of the file. You may need to change
        the permission to 'Anyone with the link', or have had many accesses.

You may still be able to access the file from the browser:

         https://drive.google.com/uc?id=1BlDBB4dLLrlN3cJhVL4nmrd_g6Jx6uP0
```

This applies to all the google drive files.

**Comment by anantarb** [comment]
Does this issue occur for triplanenet_v2_final.pth as well?

**Comment by artificialnouveau** [comment]
Yes it does

**Comment by adnan-hossain** [comment]
I am experiencing same issue as well.

**Comment by adnan-hossain** [comment]
> When I run: `source ./scripts/install_deps.sh`
> 
> I get the following errors:
> 
> ```
> python -c "import gdown; gdown.download('https://drive.google.com/uc?id=1BlDBB4dLLrlN3cJhVL4nmrd_g6Jx6uP0', 'checkpoints/pretrained/epoch_20.pth', quiet=False)"
> Access denied with the following error:
> 
>         Cannot retrieve the public link of the file. You may need to change
>         the permission to 'Anyone with the link', or have had many accesses.
> 
> You may still be able to access the file from the browser:
> 
>          https://drive.google.com/uc?id=1BlDBB4dLLrlN3cJhVL4nmrd_g6Jx6uP0
> ```
> 
> This applies to all the google drive files.

I was able to fix the issue by upgrading gdown from 4.7.1 to 4.7.3
